tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3602,7 +3603,11 @@ worldBank/4_1_SHARE_RE_IN_ELECTRICITY,Renewable electricity share of total elect
 worldBank/EG_ELC_ACCS_RU_ZS,percentage of rural population with access to electricity
 worldBank/EG_ELC_ACCS_UR_ZS,percentage of urban population with access to electricity
 worldBank/EG_ELC_ACCS_ZS,percentage of population with access to electricity
+worldBank/FR_INR_DPST,"Deposit interest rate;Deposit interest rate is the rate paid by commercial or similar banks for demand, time, or savings deposits"

```

Comment:
also, not sure if the description of what a deposit interest rate is is necessary? the more sentences we add, the harder it is to have variables be in distinct embedding spaces